### ISSUE TITLE
timeout issue for non existent resources

### ISSUE THREAD

**Issue by bblfish** [read-write-web/rww-play#65]
Non existent resources take some time to return with a timeout. Perhaps there is a way to speed the response.

**Comment by bblfish** [comment]
Sebastien, you did some research on this. Can you summarise your findings? (in case other people end up having ideas on how to improve this?)

**Comment by slorber** [comment]
Henry I have a hacky solution that currently works for this issue so maybe it's not prioritary but we need to find an elegant way to solve this problem.

Maybe it's worth asking the Akka community but I guess each container should maintain a list of all its childs so that we can query the root which will recursively pass the message to its childs unless they doesn't exist, in this case the container that doesn't have the targeted child will reply something like NotFound

Here's the hacky solution:

```
  // TODO /!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\
  // TODO remove this hardcoded method with ugly fix of timeout problems, see #65
  // TODO /!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\/!\
  // This avoids to sending no message at all because the actorSelection does not match any actor at all
  def isExistingActor(relativePath: String): Boolean = {
    val absolutePath: Path = controllers.plantain.rootContainerPath.resolve(relativePath) // TODO very bad to have plantain hardcoded here
    val exist = absolutePath.toFile.exists()
    log.warning(s"Temporary hack: will check if a file exist at $absolutePath to know if there is a corresponding actor -> answer is=$exist")
    exist
  }
```

**Comment by slorber** [comment]
I close this since a new issue was created to fijnd a proper solution
